content/blog/2022/01/mutating-policy-behave-as-validating.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Kubewarden has policies to cover all the Kubernetes Pod Security Policies.
+authors:
+	- José Guilherme Vanz
+date: 2022-01-31
+---
+
+We at the Kubewarden team has been doing an effort to allow our users to have
+policies equivalent for all the deprecated Pod Security Policies (PSP).
+So, we wrote policies to replicate the same validations available in the Kubernetes
+PSPs. During this work, we counted on the community help to map and validate
+the policies.
+
+Since Kubewarden controller <version-placeholder> and Policy Server version
+<version-placeholder>, Kubewarden users have policies which replace all the PSPs!
+All the them can be found in the Policy Hub looking for the keyword "PSP".
+
+You may are thinking: why do I need to a specific versions to have all PSP policies?
```

Comment:
```suggestion
You may be thinking: why do I need to have a specific version of all the PSP policies?
```

Reply:
Your suggestion makes me think that I need policies in specific versions. Which is not true. The components that required a minimum version is the Kubewarden controller and Policy Server. The policies version does not matter.  However, I original question is confusing as well... Let me change that.